arctic/date/_util.py

```diff
@@ -2,8 +2,7 @@
 import datetime
 import sys
 from datetime import timedelta
-
-import pandas as pd
+import pandas
```

Comment:
I'd bet the rest of the codebase will import it as `pd` - why change this just for this file?